.travis.yml

```diff
@@ -149,30 +149,38 @@ jobs:
         - go mod vendor
       script:
         - go test -v -race ./...
-        - docker build ${IMAGE_LABEL} --build-arg GOFLAGS=${GOFLAGS} -f Dockerfile -t ${REPO_NAME} .
+        - docker build ${IMAGE_LABEL} --build-arg GOFLAGS=${GOFLAGS} -f Dockerfile -t ${REPO_NAME_DRIVER} .
       before_deploy:
         - echo "$QUAY_BOT_PASSWORD" | docker login -u "$QUAY_BOT_USERNAME" --password-stdin quay.io
-        - docker tag ${REPO_NAME} ${IMAGE_FQN}
+        - docker tag ${REPO_NAME_DRIVER} ${IMAGE_FQN_DRIVER}
       deploy:
         - provider: script
-          script: docker push ${IMAGE_FQN}
+          script: docker push ${IMAGE_FQN_DRIVER}
           on:
             all_branches: true
             condition: -n "$QUAY_BOT_USERNAME" && -n "$QUAY_BOT_PASSWORD"
     
     # stage: build
-    - name: "Build - Operator: image and tag"
-      script:
+    - name: "Build - Operator: image and push to registry"
+      install:
         - sudo pip install docker
```

Comment:
do we need either of these lines, since we are using travis's docker service?

Reply:
Let me remove this line and see if it fails.

Reply:
After some discussion, I believe that this docker python sdk is NOT currently being used in our code, i tried another branch to run it through the CI and it passed... I think we can safely remove this for now.

Reply:
I'm pretty sure it isn't used so +1